src/test/java/lotto/model/RanksTest.java

```diff
@@ -0,0 +1,25 @@
+package lotto.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RanksTest {
+
+    @DisplayName("랭킹을 업데이트 한다")
+    @Test
+    void updateRanks() {
+        Ranks ranks = new Ranks();
+        MatchInfo matchInfo1 = new MatchInfo(5, true);
+        MatchInfo matchInfo2 = new MatchInfo(3, false);
+        Map<Rank, Integer> rankMap = ranks.updateRanks(
+            Arrays.asList(matchInfo1, matchInfo2)
+        );
+        assertThat(rankMap.get(Rank.SECOND)).isEqualTo(1);
+        assertThat(rankMap.get(Rank.FIFTH)).isEqualTo(1);
+        assertThat(rankMap.get(Rank.FIRST)).isEqualTo(0);
```

Comment:
`assertAll`을 이용해서 묶을 수 있을 것 같아요!

[assertAll vs multiple assertions in JUnit5](https://stackoverflow.com/questions/40796756/assertall-vs-multiple-assertions-in-junit5)

```suggestion
        assertAll(
            () -> assertThat(rankMap.get(Rank.SECOND)).isEqualTo(1),
            () -> assertThat(rankMap.get(Rank.FIFTH)).isEqualTo(1),
            () -> assertThat(rankMap.get(Rank.FIRST)).isEqualTo(0)
        );
```